composer.json

```diff
@@ -31,16 +31,17 @@
         "phpcr/phpcr": "^2.1.1",
         "phpcr/phpcr-implementation": "^2.1",
         "phpcr/phpcr-utils": "^1.3.0",
-        "doctrine/instantiator": "^1.0.1",
+        "doctrine/instantiator": "^1.0.1 || ^2.0",
         "symfony/console": "^3.4 || ^4.3 || ^5.0 || ^6.0",
-        "psr/cache": "^1.0 || ^2.0 || ^3.0"
+        "psr/cache": "^1.0 || ^2.0 || ^3.0",
+        "jackalope/jackalope-jackrabbit": "^1.4"
```

Comment:
this should not be committed. it is up to the user to chose the jackalope implementation.

```suggestion
```